espnet2/diar/frontend/default.py

```diff
@@ -0,0 +1,115 @@
+# Licensed under the MIT license.
+
+import copy
```

Comment:
Is there a reason to create diar/frontend instead of asr/frontend?

Reply:
My apology! This is something to be fixed. Thanks for pointing it out.